src/App.tsx

```diff
@@ -35,61 +36,134 @@ export function getReorderedGoods(
   const visibleGoods = [...goods];
 
   // Sort and reverse goods if needed
```

Comment:
Good job solving this task! Everything seems to be excellent, just one little thing, do not forget to remove the comments, it  is not important but make sure to delete it before push the code.